PostBuild.sh

```diff
@@ -480,12 +482,28 @@ function authselectInit {
    err_exit "Succeeded initializing authselect" NONE
 }
 
+# Disable subscription-manager
+function DisableSubscriptionManager {
+  local YUM_CONF
+  YUM_CONF=$( readlink -f /etc/yum/pluginconf.d/subscription-manager.conf )
```

Comment:
Won't the readlink will return non-zero if not found, causing `set -e` will abort?

Reply:
Ok, so, `subscription-manager` RPM is part of the `@core` RPM-group. So, it _should_ always be present (given the intended build-scenario. That said, since we notionally support using a manifest-file in place of relying on repositories containing group-metadata, probably need to test for the `subscription-manager` RPM as a hedge against that edge use-case. So, I'll re-order the function's content, some.